PWGCF/FemtoDream/Core/femtoDreamParticleHisto.h

```diff
@@ -360,10 +372,18 @@ class FemtoDreamParticleHisto
             pidTPC = part.tpcNSigmaPr();
             pidTOF = part.tofNSigmaPr();
             break;
-          case 1000010020:
+          case 1000010020: // Deuteron
             pidTPC = part.tpcNSigmaDe();
             pidTOF = part.tofNSigmaDe();
             break;
+          case 1000010030: // Triton
+            pidTPC = part.tpcNSigmaTr();
+            pidTOF = part.tofNSigmaTr();
+            break;
+          case 1000020030: // Helium3
```

Comment:
Could we move to use named constants as per
https://github.com/AliceO2Group/AliceO2/blob/dev/Common/Constants/include/CommonConstants/PhysicsConstants.h
it will be less error prone and no comments are needed

As it is commented there, the previous file expands the definitions in `<TPDGCode_8.h>` as visible in https://root.cern/doc/master/TPDGCode_8h.html

Reply:
Hi Victor,
yes. I just need to add the PID to the extra tables and some additional histograms. The framework which calculates the bitmask did already support all possible track PIDs.
Thanks for the comment. I swich the PDG codes to the name constants.